packages/bsky/src/api/app/bsky/graph/getKnownFollowers.ts

```diff
@@ -0,0 +1,122 @@
+import { mapDefined } from '@atproto/common'
+import { InvalidRequestError } from '@atproto/xrpc-server'
+import { Server } from '../../../../lexicon'
+import { QueryParams } from '../../../../lexicon/types/app/bsky/graph/getKnownFollowers'
+import AppContext from '../../../../context'
+import {
+  HydrationFnInput,
+  PresentationFnInput,
+  RulesFnInput,
+  SkeletonFnInput,
+  createPipeline,
+} from '../../../../pipeline'
+import { HydrateCtx, Hydrator } from '../../../../hydration/hydrator'
+import { Views } from '../../../../views'
+import { clearlyBadCursor, resHeaders } from '../../../util'
+
+export default function (server: Server, ctx: AppContext) {
+  const getKnownFollowers = createPipeline(
+    skeleton,
+    hydration,
+    noBlocks,
+    presentation,
+  )
+  server.app.bsky.graph.getKnownFollowers({
+    auth: ctx.authVerifier.standard,
+    handler: async ({ params, auth, req }) => {
+      const viewer = auth.credentials.iss
+      const labelers = ctx.reqLabelers(req)
+      const hydrateCtx = await ctx.hydrator.createContext({
+        labelers,
+        viewer,
+      })
+
+      const result = await getKnownFollowers(
+        { ...params, hydrateCtx: hydrateCtx.copy({ viewer }) },
```

Comment:
what's the point of the `copy` here?

Reply:
oh lawl just copied this bit from getBlocks, will remove

Reply:
oh actually forgot, it make TS happy — I want `viewer` to be defined here — that OK or is there a better way?

Reply:
ah yeah that's fine 👍